.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:2.7.15-stretch
+    steps:
+      - checkout
+      #- run:
```

Comment:
This is for our hack on osm2pgsql ... but since master osm2pgsql seems to work fine now? Either add more comments to explain this or yeah take it out.

Reply:
Yup, master seems fine now. Removed in 05f52ea.